frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -616,9 +617,24 @@ export default ({
       this.handleSendMessage(message.text, message.attachments, message.replyingMessage)
     },
     replyMessage (message) {
-      const { text, hash } = message
-      this.ephemeral.replyingMessage = { text, hash }
-      this.ephemeral.replyingTo = this.who(message)
+      const { text, hash, type } = message
+
+      if (type === MESSAGE_TYPES.INTERACTIVE) {
+        const proposal = message.proposal
+        const getNameFromMemberID = (memberID) => {
+          const profile = this.globalProfile(memberID)
+          return profile ? `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${memberID}` : L('Unknown user')
```

Comment:
Why is this check needed? I believe `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${memberID}` should be used unconditionally.

Reply:
Fixed.
(I thought the user profile data of a member who has left group would be gone and no longer captured in `ourContactProfilesById` getter. But it seems that's not the case which is a good thing.)